From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Wed, 19 Sep 2012 10:09:52 -0400
Subject: fix perl interpreter location

No need to use tricks since we know where it is
---
 config/tools/munch.pl                      | 3 +--
 config/tools/useManifestTool.pl            | 3 +--
 src/makeBaseApp/makeBaseApp.pl             | 3 +--
 src/makeBaseExt/makeBaseExt.pl             | 3 +--
 src/registry/registerRecordDeviceDriver.pl | 3 +--
 src/tools/convertRelease.pl                | 3 +--
 src/tools/cvsclean.pl                      | 3 +--
 src/tools/dos2unix.pl                      | 3 +--
 src/tools/filterWarnings.pl                | 3 +--
 src/tools/fullPathName.pl                  | 3 +--
 src/tools/replaceVAR.pl                    | 3 +--
 src/tools/useManifestTool.pl               | 3 +--
 startup/EpicsHostArch.pl                   | 3 +--
 13 files changed, 13 insertions(+), 26 deletions(-)

diff --git a/config/tools/munch.pl b/config/tools/munch.pl
index 031b98c..168f3ae 100755
--- a/config/tools/munch.pl
+++ b/config/tools/munch.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # makeConfigAppInclude.pl
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2002 The University of Chicago, as Operator of Argonne
 #     National Laboratory.
diff --git a/config/tools/useManifestTool.pl b/config/tools/useManifestTool.pl
index 8607e9c..4cf05c2 100755
--- a/config/tools/useManifestTool.pl
+++ b/config/tools/useManifestTool.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S -w  $0 ${1+"$@"}'  # -*- Mode: perl -*-
-        if 0; 
+#!/usr/bin/perl
 
 #
 # Use MS Visual C++ compiler version number to determine if
diff --git a/src/makeBaseApp/makeBaseApp.pl b/src/makeBaseApp/makeBaseApp.pl
index 7128e68..6a72b30 100755
--- a/src/makeBaseApp/makeBaseApp.pl
+++ b/src/makeBaseApp/makeBaseApp.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # makeBaseApp 
+#!/usr/bin/perl
 
 # Authors: Ralph Lange, Marty Kraimer, Andrew Johnson and Janet Anderson
 # Revision-Id: anj@aps.anl.gov-20120731184723-1d99w9yb9b7a6m1d
diff --git a/src/makeBaseExt/makeBaseExt.pl b/src/makeBaseExt/makeBaseExt.pl
index 9773bd3..4e5c3ee 100755
--- a/src/makeBaseExt/makeBaseExt.pl
+++ b/src/makeBaseExt/makeBaseExt.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # makeBaseExt 
+#!/usr/bin/perl
 
 # Authors: Ralph Lange, Marty Kraimer, Andrew Johnson and Janet Anderson
 # Revision-Id: anj@aps.anl.gov-20101005192737-disfz3vs0f3fiixd
diff --git a/src/registry/registerRecordDeviceDriver.pl b/src/registry/registerRecordDeviceDriver.pl
index 3fa64cb..587e84a 100755
--- a/src/registry/registerRecordDeviceDriver.pl
+++ b/src/registry/registerRecordDeviceDriver.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # registerRecordDeviceDriver 
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2009 UChicago Argonne LLC, as Operator of Argonne
 #     National Laboratory.
diff --git a/src/tools/convertRelease.pl b/src/tools/convertRelease.pl
index dfe6e62..06e0ea1 100644
--- a/src/tools/convertRelease.pl
+++ b/src/tools/convertRelease.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # convertRelease.pl
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2008 UChicago Argonne LLC, as Operator of Argonne
 #     National Laboratory.
diff --git a/src/tools/cvsclean.pl b/src/tools/cvsclean.pl
index d3485e7..1263f81 100644
--- a/src/tools/cvsclean.pl
+++ b/src/tools/cvsclean.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # cvsclean.pl
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2002 The University of Chicago, as Operator of Argonne
 # National Laboratory.
diff --git a/src/tools/dos2unix.pl b/src/tools/dos2unix.pl
index 64117fe..5150439 100644
--- a/src/tools/dos2unix.pl
+++ b/src/tools/dos2unix.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # makeConfigAppInclude.pl
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2002 The University of Chicago, as Operator of Argonne
 #     National Laboratory.
diff --git a/src/tools/filterWarnings.pl b/src/tools/filterWarnings.pl
index 0364c14..fdf3dd0 100644
--- a/src/tools/filterWarnings.pl
+++ b/src/tools/filterWarnings.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'      # -*- Mode: perl -*-
-	if $running_under_some_shell; # filterWarnings.pl
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2002 The University of Chicago, as Operator of Argonne
 #     National Laboratory.
diff --git a/src/tools/fullPathName.pl b/src/tools/fullPathName.pl
index 4ff31ab..025969a 100644
--- a/src/tools/fullPathName.pl
+++ b/src/tools/fullPathName.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S -w  $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if 0;
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2009 UChicago Argonne LLC, as Operator of Argonne
 #     National Laboratory.
diff --git a/src/tools/replaceVAR.pl b/src/tools/replaceVAR.pl
index 4c50f1a..fdf3fe1 100644
--- a/src/tools/replaceVAR.pl
+++ b/src/tools/replaceVAR.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # replaceVAR.pl
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2002 The University of Chicago, as Operator of Argonne
 #     National Laboratory.
diff --git a/src/tools/useManifestTool.pl b/src/tools/useManifestTool.pl
index 45986b6..968a03b 100644
--- a/src/tools/useManifestTool.pl
+++ b/src/tools/useManifestTool.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S -w  $0 ${1+"$@"}'  # -*- Mode: perl -*-
-        if 0; 
+#!/usr/bin/perl
 
 #
 # Use MS Visual C++ compiler version number to determine if
diff --git a/startup/EpicsHostArch.pl b/startup/EpicsHostArch.pl
index 3d75200..e4aa42f 100755
--- a/startup/EpicsHostArch.pl
+++ b/startup/EpicsHostArch.pl
@@ -1,5 +1,4 @@
-eval 'exec perl -S $0 ${1+"$@"}'  # -*- Mode: perl -*-
-    if $running_under_some_shell; # EpicsHostArch.pl
+#!/usr/bin/perl
 #*************************************************************************
 # Copyright (c) 2011 UChicago Argonne LLC, as Operator of Argonne
 #     National Laboratory.
